### DISCUSSION TITLE
printf format specifiers arent being colored

### DISCUSSION THREAD

**Discussion by Gaebuh** [redhat-developer/vscode-java#3269] [Q&A]
It seems that my printf format specifiers arent being colored
example:
![image](https://github.com/redhat-developer/vscode-java/assets/77945882/81149378-7971-403c-afe1-05825d78e8e9)

what it looks like when I program in c:
![image](https://github.com/redhat-developer/vscode-java/assets/77945882/231ec7bf-91ea-46d8-a28e-05993f853ea0)

edit: bump

**Comment by rgrunber** [comment]
Agreed. In fact I don't even see the newlines properly detected. This is an issue that could be supported in the TextMate grammar for string literals and text blocks. It's pretty specific but would be nice to have.

https://github.com/redhat-developer/vscode-java/blob/f598b7938c771987b0da0f85225540ed9af6ffca/language-support/java/java.tmLanguage.json#L1583-L1628
